nncf/quantization/structs.py

```diff
@@ -24,3 +25,18 @@ def __init__(self,
                  affected_insertions: List[PTTargetPoint]):
         super().__init__(quantizer_module_ref, affected_insertions)
         self.quantized_module = quantized_module
+
+
+class UnifiedScaleType(Enum):
+    """
+    UNIFY_ONLY_PER_TENSOR - only results in scale unification if per-tensor quantization is ultimately applied.
+    This is the target scenario for concat unified scales since the channel count between the concatenated tensors
+    may be mismatching and, more importantly, the concatenation might occur on exactly the channel dimension which
+    means that the concatenated tensor must reuse all quantization scales of the input per-channel
+    quantized tensors.
+    UNIFY_ALWAYS - results in scale unification for both per-channel and per-tensor quantization. This is the
+    target scenario for eltwise unified scales, as it is assumed that the eltwise ops have matching input
+    tensor shapes and therefore the quantization channel count is the same.
+    """
+    UNIFY_ONLY_PER_TENSOR = 0
```

Comment:
Good point, BTW. There can be a case when we have a DW Conv after Concat op, which means that we can quantize activations per channel and there is no need to align the inputs of Concat.